corehq/apps/app_manager/templates/app_manager/partials/forms/case_config_advanced.html

```diff
@@ -506,7 +521,7 @@ <h4 class="panel-title panel-title-nolink">
               </h4>
             </div>
             <div class="panel-body">
-                <div id="case-load-accordion" class="panel-group" data-bind="foreach: load_update_cases">
+                <div id="case-load-accordion" class="panel-group" data-bind="foreach: load_update_cases, sortableList: load_update_cases, sortableListItems: function () { return 'div:not(not_droppable)'; }">
```

Comment:
Couple of things that I don't like about this:

1. from my searching, there's not a way in knockout to use a constant, it must be wrapped in a function. Could have defined it on the model, but I like that the classes being defined are confined to this html file
2. `not(not_droppable)` is a double negative. however, if I tried to do a positive definition (`div.droppable`) it doesn't allow dropping before or after the entire list without some extra setup/adding divs, so I decided to keep it simple

Reply:
> from my searching, there's not a way in knockout to use a constant, it must be wrapped in a function

This is surprising to me, what breaks? Replacing the function with the plain string and then not doing a function call seems to work for me, but perhaps there's something I'm not testing for...?

Reply:
I think I was looking at some docs when I came to this conclusion. As long as I remove the `()` on access it works https://github.com/dimagi/commcare-hq/pull/23689/commits/c994ed16067930d7621dc1dda4df9ba39cc40597